packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1221,6 +1284,10 @@ export class TransactionController extends BaseController<
         return;
       }
 
+      if (this.inProcessOfSigning.has(transactionId)) {
+        return;
```

Comment:
Is it worth adding a `log` here to help with future debug?

Reply:
Added in https://github.com/MetaMask/core/pull/1916/commits/bd77b8d4aac1b83f8eab044143a5705fcda18205